### PR TITLE
report-issue command

### PR DESCRIPTION
**Pull request by purplecabbage**
`phonegap report-issue` : spawn the default browser where the user can edit info and submit an issue

### CODE REVIEW

File: lib/cli/report-issue.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
+
+var opener = require('opener');
+var analytics = require('./analytics');
```
Comment (filmaj) [inline lib/cli/report-issue.js]: This isn't needed, right?

Comment (purplecabbage) [inline lib/cli/report-issue.js]: Correct, thanks!  I'll fix it. 
I was originally using Gelf from analytics..,

Comment (filmaj) [inline lib/cli/report-issue.js]: Unrelated to this pull request: think it's worth tweaking the linting to pick up these things?

Comment (surajpindoria) [inline lib/cli/report-issue.js]: Go for it, sometimes it is easy to miss this stuff as the code continues changing.

**Pull request merged**